src/OrbitClientData/ModuleData.cpp

```diff
@@ -78,6 +78,16 @@ const FunctionInfo* ModuleData::FindFunctionByElfAddress(uint64_t elf_address,
   return function;
 }
 
+void ModuleData::AddFunctionInfoWithBuildId(const FunctionInfo& function_info,
+                                            const std::string& build_id) {
+  absl::MutexLock lock(&mutex_);
+  CHECK(functions_.find(function_info.address()) == functions_.end());
```

Comment:
Isn't that an absl map? In this case, I'd prefer `CHECK(!functions_.contains(...))`

Reply:
it is std::map